core/src/test/java/org/infinispan/util/ControlledConsistentHashFactory.java

```diff
@@ -113,6 +121,7 @@ public void setMembersToUse(List<Address> membersToUse) {
       this.membersToUse = membersToUse;
    }
 
+   @SerializeWith(Externalizer.class)
```

Comment:
Should we just add `ControlledConsistentHashFactory*` to the white list as you did with `BaseControlledConsistentHashFactory` above? Adds more code we have to worry about maintaining that buys us very little imo (since they are test classes).

Reply:
Or I guess that could be that Address is not Serializable?

Reply:
I have gone the other way and added Serializers for the `*ConsistentHash*` implementations. It's really internal code that shouldn't be handled by the user marshaller either way. I don't think maintenance effort is an issue, just some added boiler plate.

Reply:
yeah, I meant more it is just stuff to maintain and worry about in a test is all. I prefer when possible to have tests as specific as possible to not confuse what it is actually testing. But I understand if this has to be done.